tsfile/src/main/java/org/apache/iotdb/tsfile/read/TsFileSequenceReader.java

```diff
@@ -225,10 +226,13 @@ public String readHeadMagic() throws IOException {
    * to the end of the magic head string.
    */
   public String readHeadMagic(boolean movePosition) throws IOException {
```

Comment:
this parameter is no need ,  the tsfileInput is positioned in selfcheck already.

Reply:
Removed